### ISSUE TITLE
unexpected error

### ISSUE THREAD

**Issue by rehgar12** [conda/conda#2467]
Using Anaconda Cloud api site https://api.anaconda.org
Current conda install:

```
         platform : linux-64
    conda version : 4.0.5
```

  conda-build version : 1.20.0
       python version : 2.7.11.final.0
     requests version : 2.9.1
     root environment : /home/rap/anaconda2  (writable)
  default environment : /home/rap/anaconda2
     envs directories : /home/rap/anaconda2/envs
        package cache : /home/rap/anaconda2/pkgs
         channel URLs : https://repo.continuum.io/pkgs/free/linux-64/
                        https://repo.continuum.io/pkgs/free/noarch/
                        https://repo.continuum.io/pkgs/pro/linux-64/
                        https://repo.continuum.io/pkgs/pro/noarch/
          config file : None
    is foreign system : False

---

Traceback (most recent call last):
  File "/home/rap/anaconda2/bin/conda", line 6, in <module>
    sys.exit(main())
  File "/home/rap/anaconda2/lib/python2.7/site-packages/conda/cli/main.py", line 139, in main
    args_func(args, p)
  File "/home/rap/anaconda2/lib/python2.7/site-packages/conda/cli/main.py", line 146, in args_func
    args.func(args, p)
  File "/home/rap/anaconda2/lib/python2.7/site-packages/conda/cli/main_update.py", line 64, in execute
    install.install(args, parser, 'update')
  File "/home/rap/anaconda2/lib/python2.7/site-packages/conda/cli/install.py", line 424, in install
    plan.execute_actions(actions, index, verbose=not args.quiet)
  File "/home/rap/anaconda2/lib/python2.7/site-packages/conda/plan.py", line 539, in execute_actions
    inst.execute_instructions(plan, index, verbose)
  File "/home/rap/anaconda2/lib/python2.7/site-packages/conda/instructions.py", line 149, in execute_instructions
    cmd(state, arg)
  File "/home/rap/anaconda2/lib/python2.7/site-packages/conda/instructions.py", line 92, in LINK_CMD
    link(state['prefix'], arg, index=state['index'])
  File "/home/rap/anaconda2/lib/python2.7/site-packages/conda/instructions.py", line 88, in link
    install.link(pkgs_dir, prefix, dist, lt, index=index)
  File "/home/rap/anaconda2/lib/python2.7/site-packages/conda/install.py", line 649, in link
    _link(src, dst, lt)
  File "/home/rap/anaconda2/lib/python2.7/site-packages/conda/install.py", line 132, in _link
    shutil.copy2(src, dst)
  File "/home/rap/anaconda2/lib/python2.7/shutil.py", line 130, in copy2
    copyfile(src, dst)
  File "/home/rap/anaconda2/lib/python2.7/shutil.py", line 83, in copyfile
    with open(dst, 'wb') as fdst:
IOError: [Errno 13] Permission denied: '/home/rap/anaconda2/lib/python2.7/site-packages/numpy/__config__.py'

**Comment by ijstokes** [comment]
Do you have multiple people using the same Anaconda installation?  If so, this could be tied to #1329.

Please could you do:

```
ls -Fla /home/rap/anaconda2/lib/python2.7/site-packages/numpy/__config.py
ls -Fla ~rap/anaconda2/lib/python2.7/site-packages/ | cut -c 17-24 | sort | uniq -c
```

and send us the output?

And to solve your problem, you'll need to do:

```
sudo chown -R rap ~rap/anaconda2
```

because somehow some of your Anaconda files have ended up being owned by someone else so you can't update them.  If you want to prevent that from happening in the future (while still giving other people access to your Anaconda installation), you can do:

```
chmod -R go-w ~rap/anaconda2
chmod -R a+rX ~rap/anaconda2
```

Let us know if any of this helps.

**Comment by rehgar12** [comment]
```
rap@rap:~$ ls -Fla /home/rap/anaconda2/lib/python2.7/site-packages/numpy/__config.py
ls: cannot access /home/rap/anaconda2/lib/python2.7/site-packages/numpy/__config.py: No such file or directory
rap@rap:~$ ls -Fla ~rap/anaconda2/lib/python2.7/site-packages/ | cut -c 17-24 | sort | uniq -c
      1 
      8 ap  rap 
    323 oot root
```

That don't look good. Not sure where that config file went. I do have these guys though:
```
rap@rap:poretools$ ls /home/rap/anaconda2/lib/python2.7/site-packages/numpy/__*
/home/rap/anaconda2/lib/python2.7/site-packages/numpy/__config__.py
/home/rap/anaconda2/lib/python2.7/site-packages/numpy/__config__.pyc
/home/rap/anaconda2/lib/python2.7/site-packages/numpy/__init__.py
/home/rap/anaconda2/lib/python2.7/site-packages/numpy/__init__.pyc
```

So what I'm trying to do is get poretools working, and many of the anaconda tools are dependencies of this other tool (poretools).
If you have experience installing this program I'd appreciate your input on the following error message as well. After unpacking poretools, and changing into that unpacked directory, then running the command:

`rap@rap:poretools$ sudo python setup.py install`

some verbose stuff, then:
```
warning: no previously-included files matching '*.pyo' found anywhere in distribution
warning: no previously-included files matching '*.pyd' found anywhere in distribution
numpy/core/src/npymath/ieee754.c.src:7:29: fatal error: npy_math_common.h: No such file or directory
 #include "npy_math_common.h"
                             ^
compilation terminated.
numpy/core/src/npymath/ieee754.c.src:7:29: fatal error: npy_math_common.h: No such file or directory
 #include "npy_math_common.h"
                             ^
compilation terminated.
error: Setup script exited with error: Command "x86_64-linux-gnu-gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fPIC -Inumpy/core/include -Ibuild/src.linux-x86_64-2.7/numpy/core/include/numpy -I/usr/include/python2.7 -Ibuild/src.linux-x86_64-2.7/numpy/core/src/private -Ibuild/src.linux-x86_64-2.7/numpy/core/src/private -Ibuild/src.linux-x86_64-2.7/numpy/core/src/private -c build/src.linux-x86_64-2.7/numpy/core/src/npymath/ieee754.c -o build/temp.linux-x86_64-2.7/build/src.linux-x86_64-2.7/numpy/core/src/npymath/ieee754.o" failed with exit status 1
```

Might just have to start from scratch with all this but here's to hoping you've seen this before.
Thanks!

**Comment by kalefranz** [comment]
Permissions errors and multi-user/read-only installs have been fixed in later versions of conda 4.3.
